--- origsrc/lua-curl-0.2/src/Makefile.am	2007-09-21 11:45:36.000000000 -0500
+++ src/lua-curl-0.2/src/Makefile.am	2008-02-13 21:49:47.234375000 -0600
@@ -4,7 +4,7 @@
 		  Lua-cURL-callback.c Lua-cURL-post.c Lua-cURL-multi.c \
 		  Lua-cURL-share.c Lua-cURL.h  Lua-utility.h Lua-cURL-share.h
 				
-cURL_la_LDFLAGS = @LUA_LIBS@ @CURL_LIBS@ -avoid-version -module
+cURL_la_LDFLAGS = @LUA_LIBS@ @CURL_LIBS@ -avoid-version -module -no-undefined
 cURL_la_CFLAGS =  @LUA_CFLAGS@ @CURL_CFLAGS@
 
 libdir = @CMOD_INSTALL@
